### PR TITLE
docs: incorporate Alex feedback

### LINKED ISSUE #124: Documentation for ambiguous aliases
We should add documentation encouraging the user to input a concept id (e.g. uniprot accession) when available instead of a gene symbol to ensure an accurate normalized gene symbol is returned.

### LINKED ISSUE #220: Docs reorg
* Expand the "See the public REST service" sentence on the index page, move it up a bit (Quickstart instructions/description using live service). In general, emphasize usage of the REST endpoint.
* Collapse quick/full to combined local install page
* Clean up the README

### PR DESCRIPTION
**Pull request by jsstevenson**
Staging version of #242 

Wasn't really sure what to title this commit, just following up on some things from a 1 on 1 a while back. 

* close #220 , combine installation pages, emphasize public REST API usage, move detailed instructions out of README
* Also close #124 , add usage instructions about what kinds of search terms work best
* Add reference to @anastasiasmith1221 slides. As that work continues to develop, we can update as needed (steal figures etc)
* Add source metadata description
* Remove citations (unnecessary, though it was a helpful exercise -- if we ever need to do this again, refer back to commit history here)
* add a bit more data to the live normalize figure
* misc cleanup

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,188 +1,81 @@
...
-To use a PostgreSQL instance instead of DynamoDB, provide a PostgreSQL connection URL instead, e.g.
-
-```commandline
-export GENE_NORM_DB_URL="postgresql://postgres@localhost:5432/gene_normalizer"
-```
-
-### Adding and refreshing data
-
-Use the `gene_update` command in a shell to update the database.
+**[Documentation](https://gene-normalizer.readthedocs.io/en/latest/.html)** · [Installation](https://gene-normalizer.readthedocs.io/en/latest/install.html) · [Usage](https://gene-normalizer.readthedocs.io/en/latest/usage.html) · [API reference](https://gene-normalizer.readthedocs.io/en/latest/api/api.html)
```
Comment (korikuzma) [inline README.md:15]:
```suggestion
**[Documentation](https://gene-normalizer.readthedocs.io/en/latest/)** · [Installation](https://gene-normalizer.readthedocs.io/en/latest/install.html) · [Usage](https://gene-normalizer.readthedocs.io/en/latest/usage.html) · [API reference](https://gene-normalizer.readthedocs.io/en/latest/api/api.html)
```

File: README.md [hunk 2]
```diff
@@ -1,188 +1,81 @@
...
+        "alternate_labels": [
+            "BRAF1",
+            "B-RAF1",
+            "NS7",
+            "RAFB1",
+            "B-raf",
+            "BRAF-1"
+        ]
+    }
+    # ...
```
Comment (korikuzma) [inline README.md:54]: Schema has changed for 2x. See [here](https://github.com/cancervariants/gene-normalization/blob/staging/gene/schemas.py#L328) for an example

File: README.md [hunk 3]
```diff
@@ -1,188 +1,81 @@
...
-cd gene-normalization
-python3 -m virtualenv venv
-source venv/bin/activate
-pip install -e ".[pg,etl,dev,test]"
+```python
+>>> from gene.database import create_db
+>>> from gene.query import QueryHandler
+>>> q = QueryHandler(create_db())
+>>> result = q.normalize("KRAS")
+>>> print(result.gene_descriptor.gene_id)
```
Comment (korikuzma) [inline README.md:65]:
```suggestion
>>> print(result.normalized_id)
```

File: docs/source/index.rst [hunk 4]
```diff
@@ -14,6 +14,20 @@ Gene Normalizer |version|
...
+
+.. code-block:: pycon
+
+   >>> import requests
+   >>> result = requests.get("https://normalize.cancervariants.org/gene/normalize?q=braf")
+   >>> gene = result.json()['gene_descriptor']
+   >>> print((gene['gene_id']))
+   "hgnc:1097"
+   >>> print(gene['alternate_labels'])
+   ['B-raf', 'NS7', 'B-RAF1', 'BRAF-1', 'BRAF1', 'RAFB1']
```
Comment (korikuzma) [inline docs/source/index.rst:27]:
```suggestion
   >>> gene = result.json()['gene']
   >>> print(result.json()['normalized_id'])
   "hgnc:1097"
   >>> print(gene['aliases'])
   ['B-raf', 'NS7', 'B-RAF1', 'BRAF-1', 'BRAF1', 'RAFB1']
```

File: docs/source/install.rst [hunk 5]
```diff
@@ -0,0 +1,162 @@
...
+        "name": "gene-normalizer",
+        "url": "https://github.com/cancervariants/gene-normalization"
+      },
+      "gene_descriptor": {
+        "id": "normalize.gene:braf",
+        "type": "GeneDescriptor",
+        "label": "BRAF",
+
+        ...
+      }
```
Comment (korikuzma) [inline docs/source/install.rst:84]: Resolve like earlier comment

File: docs/source/normalizing_data/sources.rst [hunk 6]
```diff
@@ -12,7 +12,7 @@ Normalized records are constructed by aggregating data from established resource
...
+`The HUGO Gene Nomenclature Committee database <https://www.genenames.org/>`_ provides records (referred to as *Symbol Reports*) for protein-coding genes, pseudogenes, and non-coding RNA genes as designated by the HGNC. Symbol reports contain approved symbols, previously-used symbols, full names, and other relevant information for each approved gene. The Gene Normalizer extracts nomenclature, aliases, cross-references, locus types, and VRS Chromosome Locations for all HGNC gene records. Data is extracted from the latest HGNC JSON release (``hgnc_complete_set.json``) provided on the `HGNC archive page <https://www.genenames.org/download/archive/>`_. `Computed identifiers <https://vrs.ga4gh.org/en/stable/impl-guide/computed_identifiers.html>`_ are used to identify source accessions.
```
Comment (korikuzma) [inline docs/source/normalizing_data/sources.rst:15]: No more Chromosome Locations

**Review by korikuzma — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (korikuzma) [reply README.md:15]: Are we having two different RTD versions? Maybe we should point to the 2-alpha release rather than latest

Comment (jsstevenson) [reply README.md:15]: You can point to individual versions in RTD (they're keyed off of the python package version, so eg `https://gene-normalizer.readthedocs.io/en/latest/0.3.0-dev0/`) but it's a little awkward  and requires manual updating. I think most people just default point to one of the static endpoints, or operate an entirely different RTD project for different major versions (eg psycopg2 vs 3).

**Comment by jsstevenson** [comment]
Okay, I think I've caught everything...

File: README.md [hunk 7]
```diff
@@ -35,14 +35,12 @@ $ curl 'https://normalize.cancervariants.org/gene/normalize?q=BRAF' | python -m
 {
     "query": "BRAF",
     "match_type": 100,
-    "gene_descriptor": {
+    "gene": {
```
Comment (korikuzma) [inline README.md:38]:
```suggestion
    "normalized_id": "hgnc:1097",
    "gene": {
```

File: docs/source/normalizing_data/normalization.rst [hunk 8]
```diff
@@ -59,140 +60,310 @@ Normalized gene records are constructed by merging known data from all associate
 The normalized record
 ---------------------
 
-Normalized records are structured as `Genes <https://github.com/ga4gh/vrs/tree/2.0-alpha>`_. The normalized gene concept is provided and additional metadata is deposited as a label, xrefs, alternate labels, as well as Extensions for more complex information (such as loci and gene type). The following demonstrates this model for the BRAF gene:
+Normalized records are structured as `Genes <https://github.com/ga4gh/vrs/tree/2.0-alpha>`_ per the VRS 2.x schema. The normalized gene concept is provided and additional metadata is deposited as a label, xrefs, alternate labels, as well as Extensions for more complex information (such as loci and gene type). The following demonstrates this model for the BRAF gene:
```
Comment (korikuzma) [inline docs/source/normalizing_data/normalization.rst:63]:
```suggestion
Normalized records are structured as `Genes <https://github.com/ga4gh/vrs/tree/2.0-alpha>`_ per the VRS 2.x schema. The normalized gene concept is provided and additional metadata is deposited as a label, mappings, aliases, as well as Extensions for more complex information (such as loci and gene type). The following demonstrates this model for the BRAF gene:
```

File: docs/source/normalizing_data/normalization.rst [hunk 9]
```diff
@@ -59,140 +60,310 @@ Normalized gene records are constructed by merging known data from all associate
...
+       },
+       {
+         "type": "Extension",
+         "name": "ensembl_biotype",
+         "value": "protein_coding"
+       }
+     ],
+     "mappings": [
+       {
+         "id": null,
```
Comment (korikuzma) [inline docs/source/normalizing_data/normalization.rst:139]: Exclude nulls?

**Review by korikuzma — CHANGES_REQUESTED** [review]

File: docs/source/normalizing_data/normalization.rst (hunk 9 above)
Comment (jsstevenson) [reply docs/source/normalizing_data/normalization.rst:139]: Yeah, is there a reason we don't do this by default?

Comment (korikuzma) [reply docs/source/normalizing_data/normalization.rst:139]: I think I forgot to do the response_model_exclude_none in gene, but have done in other places

**Review by korikuzma — APPROVED** [review]

**Pull request merged**